.gitignore

```diff
@@ -25,3 +25,4 @@ yarn-error.log*
 .idea/
 
 frontend/src/react-app-env.d.ts
+backend/package-lock.json
```

Comment:
package lock jest dalej w gitignore

Reply:
Tak. Ale byl 2 razy 
Nic do rzeczy. Patrz na commenty wyzej

Reply:
chodzi mi o to, że nie powinien być ani razu. `package-lock.json` przyda się kiedy będziemy budować proda i będziemy chcieli mieć dokładnie te same wersje zależności co w developmencie